site/content/docs/02-template-syntax.md

```diff
@@ -210,17 +210,37 @@ If a *key* expression is provided — which must uniquely identify each list ite
 
 You can freely use destructuring patterns in each blocks.
 
-```html
-{#each items as { id, name, qty }, i (id)}
+```sv
+{#each items as {id, name, qty}, i (id)}
 	<li>{i + 1}: {name} x {qty}</li>
 {/each}
 ```
 
 ---
 
+You can use rest syntax when destructuring objects *(since [v3.2.0](https://github.com/sveltejs/svelte/blob/master/CHANGELOG.md#320))*.
+
+```sv
+{#each items as {id, ...rest}}
+	<li><span>{id}</span><MyComponent {...rest} /></li>
+{/each}
+```
+
+---
+
+You can use rest syntax when destructuring arrays *(since [v3.2.0](https://github.com/sveltejs/svelte/blob/master/CHANGELOG.md#320))*.
+
+```sv
+{#each items as [id, ...rest]}
+	<li><span>{id}</span><MyComponent values={...rest} /></li>
```

Comment:
Gotta bother you again.
```html
<MyComponent values={...rest} />
```
That's invalid syntax, you want:
```html
<MyComponent values={rest} />
```

Reply:
oooops again! I got to... rest! 🙈 😄

Reply:
:shipit: 
Rest up @mindrones 😄